src/ShareToken.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import firebase from './lib/firebase';
+import { db } from './lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
```

Comment:
You can remove a couple of imports here since they aren't in use:

```suggestion
import { db } from './lib/firebase';
```